src/types.rs

```diff
@@ -168,6 +215,8 @@ pub struct Im {
 pub enum Message {
     Standard(MessageStandard),
     BotMessage(MessageBotMessage),
+    //BotAdd(MessageBotAdd),
```

Comment:
I don't think these two lines belong in this PR.

Reply:
Gosh dangit